help/de/Help_ch_Account-Actions.xml

```diff
@@ -1205,352 +1292,344 @@
-        Payment</guibutton> button is used to add a transfer to the accounts for an interest
-        payment. If selected a dialog will be displayed to add the <xref linkend="tool-ent-int" />
-        to the account.
+      <title>Informationen zum Kontenabgleich</title>
+<!-- FIXME source: there ist a seconde, additional meaning of the button "Enter Interest charge..." -->
+      <para>Das Dialogfenster <guilabel>Informationen zum Kontenabgleich</guilabel> dient zur Eingabe des
+        <guilabel>Datum des Kontoauszugs</guilabel>, des <guilabel>Anfangssaldo</guilabel> (fixiert
+        vom letzten Abgleich) und des <guilabel>Schlusssaldo</guilabel> auf dem Kontoauszug. Die
+        Option <guilabel>Unterkonten mit einbeziehen</guilabel> wird benötigt, wenn zwei oder mehr
+        Konten verwendet werden, um ein einzelnes Bankkonto abzubilden. Die Schaltfläche
+        <guibutton>Zahlung Habenzins eingeben...</guibutton> (bei Aktivakonten) bzw.
+        <guibutton>Zahlung Sollzins eingeben...</guibutton> (bei Passivkonten) kann benutzt werden,
+        um die Buchung einer Zinszahlung für das Konto hinzuzufügen. Wenn Sie diesen Knopf
+        betätigen, erscheint ein weiteres Dialogfenster, mit dem Sie die
+        <xref linkend="tool-ent-int" /> anlegen können.
       </para>
 
       <note>
-        <para>The <guibutton>Enter Interest Payment</guibutton> button is shown only if you are reconciling an
-          account of <emphasis>Bank</emphasis> type.
+        <para>Die Schaltfläche <guibutton>Zahlung Habenzins eingeben...</guibutton> wird angezeigt, wenn Sie ein
+          Konto vom Typ <emphasis>Bank</emphasis> abgleichen.
+<!-- FIXME source: insert description of the account typ liability -->
+          Wohingegen bei abgleich eines Kontos vom Typ <emphasis>Fremdkapital</emphasis> die
+          Schaltfläche mit <guibutton>Zahlung Sollzins eingeben...</guibutton> beschriftet wird.
         </para>
       </note>
 
-      <para>When you have entered all relevant data, press <guibutton>Ok</guibutton> to proceed to the
-        <xref linkend="rec-win" /> dialog or <guibutton>Cancel</guibutton> to discard the
-        reconciliation process.
+      <para>Wenn Sie alle relevanten Daten eingegeben haben, drücken Sie <guibutton>Ok</guibutton>, um zum
+        Dialogfenster <xref linkend="rec-win" /> zu gelangen oder <guibutton>Abbrechen</guibutton>,
+        um den Abgleichvorgang zu verwerfen.
       </para>
 
       <sect3 id="tool-ent-int">
-        <title>Interest Payment</title>
-
-        <para>There are three panes in the <guilabel>Interest Payment</guilabel> dialog. At the bottom is a button
-          called <guibutton>No Auto Interest Payments for this Account</guibutton> which will cancel
-          the dialog. The top pane contains the <guilabel>Payment Information</guilabel>.
+        <title>Zinssahlung</title>
+<!-- FIXME source: The "No Auto Interest Payments for this Account" button is no longer included in the dialogue -->
```

Comment:
Stattdessen gibt es ja Zinszahlung in Kontoeigenschaften

Ist das eine Fußnote wert? Ich glaube nicht, hat ja anscheinend keiner gemerkt. ;-)